src/main/kotlin/cz/ackee/localizer/model/Localization.kt

```diff
@@ -7,7 +7,16 @@ package cz.ackee.localizer.model
 data class Localization(val resources: List<Resource>) {
 
     companion object {
-        val DEFAULT_LANGUAGES = listOf("cz", "cs", "en", "de", "sk", "pl", "es", "it", "ru", "hu", "fr", "cn")
+        val DEFAULT_LANGUAGES =
+            listOf("aa", "ab", "ae", "af", "ak", "am", "an", "ar", "as", "av", "ay", "az", "ba", "be", "bg", "bh", "bi", "bm", "bn", "bo", "br", "bs",
+                "ca", "ce", "ch", "co", "cr", "cs", "cu", "cv", "cy", "da", "de", "dv", "dz", "ee", "el", "en", "eo", "es", "et", "eu", "fa", "ff",
```

Comment:
I think that this will break current functionality because we allowed "cz" column and now we dont.

Reply:
But "cz" isn't valid language code, so it shouldn't work even now?